### DISCUSSION TITLE
Pool guru's for hire?

### DISCUSSION THREAD

**Discussion by jjjbrogan** [coinfoundry/miningcore#1618] [Q&A]
Instead of posting a mile-long string of config.json settings, errors thrown, and zcashd settings, I was curious if anyone is successfully running zcash on their MC pool - and would be available for hire for an hour or less of time.  I have BTC, BCH, and DOGE working beautifully (thank you, Oliver).  I've been trying to get ZEC going by myself, but wasting weeks of time with no success.   I have zcashd running, fully synch'd and working fine.  However, with all the config.json updates that seem perfect, I'm still errors, and I'm sure it's something I'm doing wrong. Happy to send some BTC for a review of my configs.  Thanks.

**Comment by btclinux** [comment]
You ask for help, but don't leave any contacts. Nobody can help you that way)

**Reply by jjjbrogan** [reply]
You just did.   Feel free to email me at jbrogan@jbrogan.com

**Reply by btclinux** [reply]
> You just did. Feel free to email me at [jbrogan@jbrogan.com](mailto:jbrogan@jbrogan.com)

But you don't replied :)
